### ISSUE TITLE
Remove button not visible despite having keyboard focus

### ISSUE THREAD

**Issue by MatthijsMud** [Yuvaleros/material-ui-dropzone#190]
### Describe the bug
The remove button does not become visible when it receives keyboard focus (unless the cursor is also hovering over it).

### Steps to reproduce
1. Upload an image
1. Press `Tab` on keyboard

### Expected behavior
The interface element that has keyboard focus to be visible, and highlighted.

### Versions

- OS: Observed on Windows and Linux Mint
- Browser: Observed on FireFox, Chrome and Edge
